package.json

```diff
@@ -41,10 +39,13 @@
   },
   "homepage": "https://github.com/asciidoctor/asciidoctor-reveal.js",
   "dependencies": {
-    "asciidoctor.js": "1.5.9",
     "reveal.js": "3.7.0"
   },
+  "peerDependencies": {
+    "@asciidoctor/core": "^2.0.0"
+  },
```

Comment:
This move is following this discussion: https://github.com/asciidoctor/asciidoctor-reveal.js/issues/187

Reply:
I think this should be documented as a comment. As a casual npm user, the added complexity of peer dependency is unclear. Something should be written to document that choice. I'm thinking specifically of the relationship with Opal versioning here. Some dev docs should need updating too.